superset-frontend/package.json

```diff
@@ -19,10 +19,10 @@
     "build-instrumented": "cross-env NODE_ENV=development BABEL_ENV=instrumented webpack --mode=development --color",
     "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production BABEL_ENV=\"${BABEL_ENV:=production}\" webpack --mode=production --color",
     "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx . && npm run type",
-    "prettier-check": "prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'",
+    "prettier-check": "prettier --check '{src}/**/*.{css,less,sass,scss}'",
     "lint-fix": "eslint --fix --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx . && npm run clean-css && npm run type",
-    "clean-css": "prettier --write '{src,stylesheets}/**/*.{css,less,sass,scss}'",
-    "format": "prettier --write './{src,spec,stylesheets,cypress-base}/**/*{.js,.jsx,.ts,.tsx,.css,.less,.scss,.sass}'",
+    "clean-css": "prettier --write '{src}/**/*.{css,less,sass,scss}'",
+    "format": "prettier --write './{src,spec,cypress-base}/**/*{.js,.jsx,.ts,.tsx,.css,.less,.scss,.sass}'",
```

Comment:
The cases with `'{src}/**/...'` would probably look nicer as just `'src/**/...'`

Reply:
That was the thing making the linter fail. I changed to your suggestion.